style-sheets/landing-page-components-overrides.scss

```diff
@@ -0,0 +1,163 @@
+@use '../design-tokens/colors.scss' as colors;
+
+/* ------------------------------- Landing Page ------------------------------- */
+
+.row {
+    margin-left: auto !important;
+    padding-left: 1rem !important;
+}
+
+.logo {
+    margin-top: 2.5rem;
+}
+.full-width {
+    max-width: 100%;
+    overflow: hidden;
+}
+.landing-grid {
+    padding-right: 0;
+    height: 100vh;
+}
+
+.landing-title {
+    margin-top: 4.375rem;
+    font-size: 4.75rem;
+    font-weight: 400;
+    line-height: 5.375rem;
+}
+
+.landing-subtitle {
+    margin-top: 0.5rem;
+    color: colors.$blue-60;
+    font-size: 2rem;
+    font-weight: 400;
+    line-height: 2.5rem;
+}
+
+.landing-desc {
+    margin-top: 2.5rem;
+    color: colors.$gray-70;
+    font-size: 2rem;
+    font-weight: 400;
+    line-height: 2.5rem;
+    margin-bottom: 3.75rem;
+}
+
+.btn-landing {
+    width: 18rem;
+    height: 3rem;
+    text-align: inherit;
+    margin-right: 0.625rem !important;
+    font-size: var(--bcgov-body-compact-01-font-size, 0.875rem);
```

Comment:
these "--bcgov-body-compact-01-font-size" looks like some variable from FAM, can we find what it is and if we could use the design token for these?

Reply:
@MCatherine1994 @NickSaglioni  Token for "body-compact-01" in the design system is $body-compact-01. The token should be inheriting these properties:  
![Capture-body-token](https://user-images.githubusercontent.com/121979429/235480149-a04794fc-654d-446f-8a72-a303be01d140.PNG).

Here is the reference link for the body type in Figma: https://www.figma.com/file/KDXuYanyOsBCM5vpPmNvGf/FSA-Design-System?node-id=11227-270859&t=dNX6YoB5RpIasQjk-4